docs/model_customization.md

```diff
@@ -0,0 +1,61 @@
+# Model Customization
+
+By default, torchchat (and PyTorch) defaults to unquantized [eager execution](https://pytorch.org/blog/optimizing-production-pytorch-performance-with-graph-transformations/).
+
+This page goes over the different options torchchat provides for customizing the model execution for inference.
+- Device
+- Compilation
```

Comment:
This doesn't fit very well IMHO in this otherwise awesome high level, almost orthogonal optimization knob categorization. It could be just me.

Reply:
Agreed, it's not an optimization knob, it's a model customization knob

Quant/optimization gets it's own page